src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -594,10 +594,53 @@ public unsafe void Reset()
         {
             _updateStopwatch.Stop();
             _renderStopwatch.Stop();
-            _glfw.DestroyWindow(_windowPtr);
+
+            try
+            {
+                _glfw.DestroyWindow(_windowPtr);
+            }
+#pragma warning disable 168
+            // If the window is already destroyed, it throws an exception,
+            // but we want the window destroyed anyways, so just ignore it
+            catch(GlfwException _e)
+#pragma warning restore 168
+            {
+                

```

Comment:
Should probably move the comment into the exception block so Rider doesn't kick up a fuss.